build.sh

```diff
@@ -1,7 +1,27 @@
 #!/usr/bin/env bash
 
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
+
+# Stop script if subcommand fails
+set -e
+
 source="${BASH_SOURCE[0]}"
 
+bootstrap=false
+
+args=""
+while [[ $# > 0 ]]; do
+  opt="$(echo "${1/#--/-}" | tr "[:upper:]" "[:lower:]")"
+  case "$opt" in
+    -integrationtest)
+      bootstrap=true
+      ;;
+  esac
+  args="$args $1"
+  shift
+done
+
```

Comment:
I'm no bash expert, but I don't see how the `bootstrap` value makes it into the rest of the build?
Maybe it's intentional that this only works on Windows? (I would very much vote against doing that, but maybe for now it's OK).

Reply:
I can revert this change -- I copied it over from my other bootstrap change and found a simpler way, so we no longer need it.